roles/openldap-remove-password-policy/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: Run remove policy for service users
+  command: "{{openldap_password_policy.config_dir}}password_policy_util.sh -r {{ item.name }}"
```

Comment:
for readability the joining slash for paths like this should be in the string,  not in the variable.    e.g. `{{openldap_password_policy.config_dir}}/password_policy_util.sh`